### ISSUE TITLE
Unconsistency in scatter function

### ISSUE THREAD

**Issue by cp1972** [erdogant/clustimage#7]
Hello again, I think that we have a little inconsistency in the scatter function, leading the inconsistent behavior -- in my case, two inconsistent behaviors: 1. some scatter plots will show up and suddenly disappear, 2. no scatter plot shows up. Here is a minimal working example to reproduce these inconsistencies: 

import sys
import os
import glob
import numpy as np
from clustimage import Clustimage
import matplotlib.pyplot as plt

cl = Clustimage(method='pca',dirpath=None,embedding='tsne',grayscale=False,dim=(128,128),params_pca={'n_components':0.95})
in_files = input("""Give the absolute path to a directory with your files: \n""")
some_files = glob.glob(in_files)
results = cl.fit_transform(some_files,
                           cluster='agglomerative',
                           evaluate='silhouette',
                           metric='euclidean',
                           linkage='ward',
                           min_clust=3,
                           max_clust=6,
                           cluster_space='high')

cl.clusteval.plot()
cl.clusteval.scatter(cl.results['xycoord'])
cl.pca.plot()
cl.plot_unique(img_mean=False) 
cl.plot(cmap='binary')
cl.scatter(zoom=1, img_mean=False)
cl.scatter(zoom=None, dotsize=200, figsize=(25, 15), args_scatter={'fontsize':24, 'gradient':'#FFFFFF', 'cmap':'Set2', 'legend':True})

**What I have done**: your scatter function returns fig, ax which is a tuple of plt.subplot; plotting fig makes the plot showing up and disappearing all of a sudden. As a result, you can not save some scatter plots. 

**Workaround**: 

cl.scatter(zoom=None, dotsize=200, figsize=(25, 15), args_scatter={'fontsize':24, 'gradient':'#FFFFFF', 'cmap':'Set2', 'legend':True})
plt.show()

With plt.show(), the scatter plots wait for you to close the gui.

Best wishes cp

**Comment by erdogant** [comment]
Thank you for mentioning this. 
I updated the scatter function and added plt.show() and plt.draw() to make sure it does not disappear.
Update with: `pip install -U clustimage`

**Comment by erdogant** [comment]
I am closing this one. Re-open this issue if still not fixed.
